src/main/java/org/galatea/starter/AppConfig.java

```diff
@@ -4,17 +4,25 @@
 import net.sf.aspect4log.aspect.LogAspect;
 import org.galatea.starter.domain.SettlementMission;
 import org.galatea.starter.service.IAgreementTransformer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.CacheManager;
 import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.ehcache.EhCacheCacheManager;
+import org.springframework.cache.ehcache.EhCacheManagerFactoryBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.core.io.ClassPathResource;
 
 @Slf4j
 @Configuration
 @EnableAspectJAutoProxy
 @EnableCaching
 public class AppConfig {
 
+  @Value("${cache-config}")
+  public String cacheConfig;
```

Comment:
can we move this to not be a member variable, and instead do an @Value on a method parameter to the @Bean ehCacheCacheManager() method

Reply:
Fixed